src/pages/home/report/ReportActionsView.js

```diff
@@ -206,6 +208,25 @@ class ReportActionsView extends React.Component {
         this.recordMaxAction();
     }
 
+    /**
+     * This function overrides the CellRendererComponent (defaults to a plain View), so that the active ReportActionItem
+     * can be given a higher z-index than the others. This prevents issues where content overlapping between rows
+     * is hidden beneath other rows.
+     *
+     * @param {Object} index - The ReportAction item in the FlatList.
+     * @param {Object|Array} style – The default styles of the CellRendererComponent provided by the CellRenderer.
+     * @param {Object} props – All the other Props provided to the CellRendererComponent by default.
+     * @returns {React.Component}
+     */
+    renderCell({item, style, ...props}) {
+        const cellStyle = [
+            style,
+            {zIndex: item.action.sequenceNumber === this.state.activeReportActionID ? 1 : 0},
+        ];
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        return <View style={cellStyle} {...props} />;
```

Comment:
Sounds convoluted, but one idea would be to make this a new component and then subscribe to `activeReportActionID` as an Onyx prop. This component should also only re-render if that prop changes. That might fix a performance regression while scrolling that I'm seeing. 

But maybe there's a simpler solution...

Reply:
Simpler solution was to just use the sequenceNumber for the `zIndex` so each row has a higher sequence number than the row above.